neqo-common/src/qlog.rs

```diff
@@ -205,15 +221,14 @@ mod test {
     }
 
     #[test]
-    fn add_event() {
+    fn add_event_with_instant() {
         let (log, contents) = test_fixture::new_neqo_qlog();
-        log.add_event(|| Some(Event::with_time(1.1, EV_DATA)));
+        log.add_event_with_instant(|| Some(Event::with_time(0.0, EV_DATA)), Instant::now());
         assert_eq!(
-            contents.to_string(),
-            format!(
-                "{EXPECTED_LOG_HEADER}{e}",
-                e = EXPECTED_LOG_EVENT.replace("\"time\":0.0,", "\"time\":1.1,")
-            )
+            Regex::new("\"time\":[0-9].[0-9]*,")
+                .unwrap()
+                .replace(&contents.to_string(), "\"time\":0.0,"),
+            format!("{EXPECTED_LOG_HEADER}{EXPECTED_LOG_EVENT}"),
```

Comment:
No way to avoid using `regex`?

Reply:
Yes, but with additional complexity, basically reinventing the wheel.

Given that we already use `regex` elsewhere and given that this is a test, thus requiring `regex` as a dev-dependency only, I consider this the cleaner option.

Happy to implement an alternative if you feel strongly about it @larseggert.